### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @aeotero

### CODE REVIEW

**Comment by aeotero** [comment]
@faloi  termine con esto

File: plagas.iml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (faloi) [inline plagas.iml:1]: Me intriga qué te habrá mostrado el Intellij de un proyecto Wollok. :thinking:

File: README.md [hunk 2]
```diff
@@ -22,4 +22,5 @@ Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archi
 
 ![Directamente desde un link](http://www.plantuml.com/plantuml/png/JSqn2W9138NXFgT8gLUmMb5hlS9Z9gu3cGmaMH74kxl25lR_zzuK2S-bAnL51DyKC_XDpEcmw2tbOdVnBXLrjp_mKUT17zhoqrBWA0y4ool5wl-2jQVe6TR63rxJnuQdDLmllm00)
 
-![Desde un archivo local](./images/ejemplo-uml.png)
+![Desde un archivo local](./images/DiagramaDeClases_Plagas.png)
```
Comment (faloi) [inline README.md:25]: Ok, aunque dos cosas:
* no incluimos getters y setters;
* una línea para dividir atributos de métodos facilita la lectura (y además, está en el estándar :roll_eyes:).

File: src/barrio.wlk [hunk 3]
```diff
@@ -3,20 +3,25 @@ class Barrio {
 	const property elementos = []
 
 	method agregarElemento(elemento) {
+	#Param: obj TDA "elemento"
```
Comment (faloi) [inline src/barrio.wlk:6]: Está bien, pero lo de TDA acá lo llamamos clases.

**Pull request closed without merging**